netmanager-app/app/(authenticated)/clients/page.tsx

```
@@ -0,0 +1,156 @@
+import { DataTable } from "./data-table"
+import { columns } from "./columns"
+import { ActivateClientDialog, DeactivateClientDialog } from "./dialogs"
+import { getClientsApi, activateUserClientApi } from "@/core/apis/analytics"
+import { useToast } from "@/components/ui/use-toast"
+import type { Client } from "@/app/types/clients"
+import withPermission from "@/app/pageAccess"
+
+const formatDate = (dateString: string | undefined): string => {
+  if (!dateString) return "N/A"
+  const date = new Date(dateString)
+  if (isNaN(date.getTime())) return "Invalid Date"
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)